src/shared/components/simpleTable/styles.scss

```diff
@@ -5,6 +5,14 @@ th {
   background-color: rgb(255,255,255); // solid background
 }
 
+tbody>tr:nth-of-type(odd).highlight {
+  background-color: #E0E000 !important;
+}
+
 tbody>tr:nth-of-type(even) { // solid background - hard coded hack, react-bootstrap .table-striped makes ODD # rows grey
   background-color: rgb(255,255,255);
+}
+
+tbody>tr:nth-of-type(even).highlight {
+  background-color: #E9E900;
```

Comment:
i think best to avoid !important if possible because then you can't override it!!